src/ngSanitize/sanitize.js

```diff
@@ -200,6 +200,52 @@ function $SanitizeProvider() {
     }
   };
 
+
+  /**
+   * @ngdoc method
+   * @name $sanitizeProvider#addValidElements
+   * @kind function
+   *
+   * @param {Array|Object} elements List of valid elements.
+   */
+  this.addValidElements = function(elements) {
+    var _validElements = [];
+
+    if (angular.isArray(elements)) {
+      _validElements = elements;
+    } else if (angular.isObject(elements)) {
+      angular.forEach(['htmlVoidElements', 'htmlElements', 'svgElements'], function(elementsType) {
+        if (!elements[elementsType]) {
+          return;
+        }
+
+        if (elementsType === 'svgElements') {
+          angular.extend(svgElements, toMap(elements[elementsType].join(',')));
```

Comment:
Extending `svgElements` has no effect if called after the service has been instantiated. This can be unintuitive (because theoretically it is possible to use provider methods after a service has been instantiated and in many case everything works as expected)

I think it would be better to either throw or make it a no-op after instantiation (and document that).

Reply:
As I see `enableSvg` also doesn't work after instantiation.

Reply:
@gkalpak 

> I think it would be better to either throw or make it a no-op after instantiation (and document that).

How can I check that service has been instantiated? (without additional variable)

Reply:
You can't (without additional variable) :grin: